src/javascripts/components/back-to-top.js

```diff
@@ -5,6 +5,11 @@ function BackToTop ($module) {
 }
 
 BackToTop.prototype.init = function () {
+  var $module = this.$module
+  if (!$module) {
+    return
+  }
+
```

Comment:
Given we don't re-use `$module` anywhere else in the function (we use `this.$module` directly) can we avoid the assignment?

```suggestion
  if (!this.$module) {
    return
  }

```